app/templates/main/browse.html

```diff
@@ -340,7 +340,7 @@ <h1 class="govuk-heading-l browse__records-found__text">Records found {{ num_rec
                                                 {% if record['closure_period'] %}
                                                     {{ record['closure_period'] }} years
                                                 {% else %}
-                                                    -
+                                                    None
```

Comment:
Do we know if we want to show None or - for empty fields? Can we check with Terry / his designs?

If there is no decision atm I'm happy for you to go with whatever you prefer

Reply:
I have kept the output consistent for all fields to show 'None' for now, There is a new ticket [AYR-608 check consignment ref view- without file meta data] has been created to discuss with Terry and Andrew on the design

Reply:
I'd suggest not applying the styling on the status field if it is None, but not a big deal since we never expect that case